src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -201,254 +253,259 @@ const ProfileHeading = ({ ownerID }) => {
+                    <Grid component={Hidden} item lgUp xs={12}>
+                        <ResponsiveButton
+                            component={Link}
+                            to={`/form/${userInfo.accountType.toLowerCase()}/${userInfo.user.id}`}
+                            variant="outlined"
+                        >
+                            <EditIcon />
+                        </ResponsiveButton>
+                    </Grid>
+                </>
+            )}
         </Grid>
-      </>
     );
 
-    const EmailRow = () => (
+    const ResetDialog = () => (
       <>
-        <Grid className="emailPadding" item md={1}>
-          <a href={`mailto:${userInfo.user.email}`}>
-            <EmailIcon />
-          </a>
-        </Grid>
-        <Grid className="emailPadding" item md={5}>
-          <a href={`mailto:${userInfo.user.email}`}>
-            <Typography className="rowText">{userInfo.user.email}</Typography>
-          </a>
-        </Grid>
+        <Dialog
+          open={open}
+          onClose={handleClosePassword}
+          maxWidth="xs"
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle disableTypography id="alert-dialog-title" className="center dialog-padding">
+            Do you want to reset this user's password?
+          </DialogTitle>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description" className="center">
+              When you reset the password, a randomly generated password will
+              become the user's new password. This action cannot be reverted. An
+              automatic email will be sent out notifying the user that their
+              password has been reset. The user can also reset their own password
+              through the portal login page.
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions className="dialog-actions">
+            <ResponsiveButton
+              onClick={handleClosePassword}
+              variant="outlined"
+              className="cancelBtn"
+            >
+              cancel
+            </ResponsiveButton>
+            <ResponsiveButton
+              onClick={handleClosePasswordReset}
+              className="resetBtn"
+              autoFocus
+            >
+              reset password
+            </ResponsiveButton>
+          </DialogActions>
+        </Dialog>
+        <Dialog
+          open={openReset}
+          onClose={handleCloseReset}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle disableTypography id="alert-dialog-title" className="center">
+            Password has been reset.
+          </DialogTitle>
+          <DialogContent className="center dialog-padding">
+            <DialogContentText id="alert-dialog-description">
+              The new password for {fullName(userInfo.user)} ID #
```

Comment:
When you move this component out of ProfileHeading be sure to take in the userInfo as a prop

Reply:
Thank you for the reminder!